.coveragerc

```diff
@@ -2,6 +2,7 @@
 omit =
     tests/*
     venv2/*
+    venv2.7.18/*
```

Comment:
Do we really need to omit those folders in coveragerc? I never had a problem with them in the past, but maybe, it was because my venv folders were all hidden, like `.venv`

Reply:
It was causing problems in github ci, for some reason it was collecting from the venv folder :shrug: this was easiest

Reply:
Ah, I see. Yeah, coverage will likely gather metrics from venv folders if they are present. I remember there is a workaround for that (maybe creating then with .venv*?). But, thinking of it now, why do we need a virtualenv during CI execution? I would expect we could just `pip install -r requirements.txt` and be done with it. 

The advantage of installing the packages without a virtualenv is that we can cache the pip folder for other executions and make it slightly faster.

Reply:
yeah I did not call it inside virtualenv in CI in the end... so now It works in CI as expected and for local runs inside vritual venv I kept `venv*` in coveragerc